transaction.go

```diff
@@ -6,23 +6,12 @@ import (
 	"fmt"
 )
 
-// Transaction executes txFunc within a database transaction that is passed in to txFunc as tx Connection.
-// Transaction returns all errors from txFunc or transaction commit errors happening after txFunc.
-// If parentConn is already a transaction, then it is passed through to txFunc unchanged as tx Connection
-// and no parentConn.Begin, Commit, or Rollback calls will occour within this Transaction call.
-// An error is returned, if the requested transaction options passed via opts
-// are stricter than the options of the parent transaction.
+// IsolatedTransaction executes txFunc within a database transaction that is passed in to txFunc as tx Connection.
+// IsolatedTransaction returns all errors from txFunc or transaction commit errors happening after txFunc.
+// If parentConn is already a transaction, a brand new transaction will begin on the parent's connection.
 // Errors and panics from txFunc will rollback the transaction if parentConn was not already a transaction.
```

Comment:
Since we don't inherit the parent transaction, errors will always roll back the current transaction

Reply:
Fixed in https://github.com/domonda/go-sqldb/commit/077f82724f22715564a6c1f4883d905e1be9d5bb.